app/models/compute_resources/foreman/model/ovirt.rb

```diff
@@ -576,14 +576,27 @@ def create_interfaces(vm, attrs)
     def create_volumes(vm, attrs)
       # add volumes
       volumes = nested_attributes_for :volumes, attrs
-      volumes.map do |vol|
+      volumes.each do |vol|
+        next if vol[:id].blank?
```

Comment:
Shouldn't it be the other way around?
```suggestion
        next if vol[:id]
```
We are creating only new volumes here, aren't we?

Reply:
Nope, it should stay the same: https://github.com/theforeman/foreman/blob/develop/app/models/compute_resources/foreman/model/ovirt.rb#L582

the reason is that the volume id wasn't created yet.

Reply:
Here you are setting the condition when to  _skip_ interface creation.

It should be the opposite of the condition you had:
Original code: create an interface if `:id` is blank
This PR: skip interface creation if `:id` is blank.